aten/src/ATen/native/TensorShape.cpp

```diff
@@ -350,6 +350,38 @@ namespace native {
 DEFINE_DISPATCH(cat_serial_stub);
 DEFINE_DISPATCH(stack_serial_stub);
 
+
+namespace {
```

Comment:
Should we make this reusable by `CompositeViewCopyKernels.cpp`?

Reply:
yes, wasnt sure where to put the util functions

Reply:
So I guess it's cleaner to just autogen all these kernels if possible. It seems like most of them are having the same logic as the generated ones.

Reply:
Just put them in a header. If you're not sure what header to put them in, make a fresh one! ViewCopy.h maybe?